myft/ui/save-article-to-list-variant.js

```diff
@@ -0,0 +1,278 @@
+	}
+
+	createListOverlay.open();
+	createListOverlay.wrapper.addEventListener('oOverlay.ready', (data) => {
+		realignListener(data.currentTarget);
+
+		if (lists && lists.length) {
+			const listElement = ListsElement(lists, addToList, removeFromList);
+			const overlayContent = document.querySelector('.o-overlay__content');
+			overlayContent.insertAdjacentElement('afterbegin', listElement);
+		}
+
+		contentElement.addEventListener('click', openFormHandler);
+
+		document.querySelector('.article-content').addEventListener('click', outsideClickHandler);
+	});
+
+	createListOverlay.wrapper.addEventListener('oOverlay.destroy', () => {
+		const tooltipTemplate = document.createElement('div');
+		const opts = {
+			target: 'o-header-top-link-myft',
+			content: 'Go to saved articles in myFT to find your lists',
+			showOnConstruction: true,
+			closeAfter: 5,
+			position: 'below'
+		};
+
+		new oTooltip(tooltipTemplate, opts);
+
+		contentElement.removeEventListener('click', openFormHandler);
+
+		document.querySelector('.article-content').removeEventListener('click', outsideClickHandler);
+	});
+
+	window.addEventListener('scroll', function () {
+		realignListener(createListOverlay.wrapper, window.scrollY);
+	});
+
+	window.addEventListener('oViewport.resize', () => {
+		realignListener(createListOverlay.wrapper);
+	});
+}
+
+function stringToHTMLElement (string) {
+	const template = document.createElement('template');
+	template.innerHTML = string.trim();
+	return template.content.firstChild;
+}
+
+function FormElement (createList) {
+	const formString = `
+	<form class='myft-ui-create-list-variant-form'>
+		<label class="o-forms-field">
+			<span class="o-forms-input o-forms-input--text">
+					<input type="text" name="list-name">
+			</span>
+		</label>
+		<button class="o-buttons o-buttons--secondary" type="submit">
+			Save
+		</button>
+	</form>
+	`;
+
+	const formElement = stringToHTMLElement(formString);
+
+	formElement.querySelector('button[type="submit"]').addEventListener('click', (e) => {
+		e.preventDefault();
+		e.stopPropagation();
+		const inputListName = formElement.querySelector('input[name="list-name"]');
+		createList(inputListName.value);
+		inputListName.value = '';
+		formElement.remove();
+	});
+
+	return formElement;
+}
+
+function ContentElement () {
+	let content = `
+		<div class="myft-ui-create-list-variant-footer">
+			<button class="myft-ui-create-list-variant-add">Add to a new list</button>
+			${!lists.length ? '<p class="myft-ui-create-list-variant-add-description">Lists are a simple way to curate your content</p>' : ''}
+		</div>
+	`;
+
+	return stringToHTMLElement(content);
+}
+
+function HeadingElement () {
+	const heading = `
+		<span class="myft-ui-create-list-variant-heading">Added to <a href="https://www.ft.com/myft/saved-articles">saved articles</a> in <span class="myft-ui-create-list-variant-icon"><span class="myft-ui-create-list-variant-icon-visually-hidden">my ft</span></span></span>
```

Comment:
```suggestion
		<span class="myft-ui-create-list-variant-heading">Added to <a href="https://www.ft.com/myft/saved-articles">saved articles</a> in <span class="myft-ui-create-list-variant-icon"><span class="myft-ui-create-list-variant-icon-visually-hidden">my FT</span></span></span>
```

This allows VoiceOver to read the characters correctly